lmp/build.sh

```diff
@@ -33,9 +33,13 @@ cat /root/.gitconfig >>  /home/builder/.gitconfig
 cp /root/.netrc /home/builder/.netrc || true
 
 # Detect the base LmP version we are building on
+# A user may have place arbitrary tags into their lmp-manifest and we don't
+# want to accidentally pick one of them up. Git doesn't really distinguish
+# were tags come from, so:
+git --git-dir .repo/manifests.git tag | xargs git --git-dir .repo/manifests.git tag -d > /dev/null
```

Comment:
Why this git operations don't use the helper `run` like the others?

Reply:
it was just subjective. "run" shows the command and ouput but i'm not sure it works great with a piped sequence. i think there's enough logic stacking up here, that I'm going to make a shell function to help contain it.